docs/dyndns/configure.rst

```diff
@@ -174,10 +174,15 @@ expected.
 
 Updating Multiple Domains
 `````````````````````````
-To update multiple domain or subdomains, it is best to designate one of them
-as the main domain, and create CNAME records for the others, so that they act
-as DNS aliases for the main domain.
-You can use do that either via the web interface or the API.
-
-If you try to update several subdomains directly (by issuing multiple update
-requests), your update requests may be refused (see :ref:`rate-limits`).
```

Comment:
I'd like to retain the reference to rate limiting if you try updating using individual requests.